### DISCUSSION TITLE
Disabling packet capture

### DISCUSSION THREAD

**Discussion by thekibiru03** [Security-Onion-Solutions/securityonion#2132] [Unsupported Versions] [answered]
Hello.
I am running SO 2.3 in a home environment where packet capture is not critical due to disk space constraints.
How can I disable packet capture? I have tried stopping the **so-steno** docker container but it restarts after a while.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/release-notes.html#changes:
> Stenographer can now be disabled on a sensor node by setting the pillar steno:enabled:false in its minion.sls file or globally if set in the global.sls file
